### PR TITLE
Implement `Display` for `ConfigId`

### PR DESCRIPTION
**Pull request by samdealy**

### CODE REVIEW

**Comment by samdealy** [comment]
Current dependencies on/for this PR:
* main
  * **PR #308** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/308" target="_blank">Graphite</a> 
    * **PR #310** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/310" target="_blank">Graphite</a> 
      * **PR #311** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/311" target="_blank">Graphite</a> 
        * **PR #312** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/312" target="_blank">Graphite</a> 
          * **PR #313** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/313" target="_blank">Graphite</a> 
            * **PR #314** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/314" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/314?utm_source=stack-comment).

File: core/types/src/config_id.rs [hunk 1]
```diff
@@ -1,21 +1,47 @@
...
 impl Default for ConfigId {
     fn default() -> Self {
         Self::from([0; SGX_CONFIGID_SIZE])
     }
 }
+
+#[cfg(test)]
+mod tests {
+    extern crate std;
+    use crate::ConfigId;
```
Comment (nick-mobilecoin) [inline core/types/src/config_id.rs:34]: :wrench: Doesn't matter too much, but usually tests do
```suggestion
    use super::*;
```

**Review by nick-mobilecoin — APPROVED** [review]

File: core/types/src/config_id.rs [hunk 2]
```diff
@@ -1,21 +1,47 @@
...
+mod tests {
+    extern crate std;
+
+    use super::*;
+    use std::format;
+
+    #[test]
+    fn display_config_id() {
+        let sgx_config_id_t = [34u8; 64];
+        let config_id = ConfigId::from(sgx_config_id_t);
```
Comment (jcape) [inline core/types/src/config_id.rs:40]: Nit: don't name variables the same as types...
```suggestion
        let inner = [34u8; 64];
        let config_id = ConfigId::from(inner);
```

**Review by jcape — APPROVED** [review]
Approved with nit about variable naming.

**Pull request closed without merging**